__tests__/schema/opportunity.ts

```diff
@@ -337,3 +345,136 @@ describe('opportunity queries', () => {
     });
   });
 });
+
+describe('query getCandidatePreferences', () => {
+  const QUERY = /* GraphQL */ `
+    query GetCandidatePreferences {
+      getCandidatePreferences {
+        status
+        cv {
+          blob
+          bucket
+          lastModified
+        }
+        role
+        roleType
+        salaryExpectation {
+          min
+          period
+        }
+        location {
+          city
+          country
+          subdivision
+          continent
+          type
+        }
+        locationType
+        employmentType
+        companySize
+        companyStage
+      }
+    }
+  `;
+
+  beforeEach(async () => {
+    await saveFixtures(con, UserCandidatePreference, [
+      {
+        userId: '1',
+        role: 'Full Stack Developer',
+        salaryExpectation: { min: 50000, period: SalaryPeriod.ANNUAL },
+        location: [{ country: 'Norway' }],
```

Comment:
Probably good to make a bigger object, just incase something like multiples don't return.